modules/CmpdReg/src/client/ReparentLot.coffee

```diff
@@ -0,0 +1,250 @@
+		
+	handleBackToCregButtonClicked: ->
+		window.location.href = 	window.configuration.serverConnection.baseServerURL
+		
+	getUlFromCodeArray: (codeArray, codeKey, nameKey, link) ->
+		ul = "<ul>";
+		_.each(codeArray, (code) ->
+			descriptionText = ""
+			if code.description?
+				descriptionText = ": #{code.description}"
+			if link?
+				# target blank
+				if code[codeKey] == code[nameKey] 
+					# target blank a tag with a href to link with code
+					ul += "<li><a href='#{link+code[codeKey]}' target='_blank'>#{code[codeKey]}#{descriptionText}</a></li>"
+				else
+					# target blank a tag with a href to link with code
+					ul += "<li><a href='#{link+code[codeKey]}' target='_blank'>#{code[codeKey]} \"#{code[nameKey]}\"</a>#{descriptionText}</li>"
+			else 
+				if code[codeKey] == code[nameKey]
+					ul += "<li>#{code[codeKey]}#{descriptionText}</li>"
+				else
+					ul += "<li>#{code[codeKey]} \"#{code[nameKey]}\"#{descriptionText}" + "</li>"	
+		);
+		ul += "</ul>";
+		return ul;
+		
+	summarizeLinkedCodeTable: (codeArray, header, link) ->
+		# Get linked experiments summary
+		entitySummary = ""
+		hasLinkedEntity = codeArray? && codeArray.length > 0
+		if hasLinkedEntity
+			if codeArray[0].containerBarcode?
+				codeKey = "containerBarcode"
+				nameKey = "wellName"
+			else
+				codeKey = "code"
+				nameKey = "name"
+			linkedEntities = _.sortBy(codeArray, (codeObject) -> codeObject[codeKey])
+			entitySummary += "<li>#{header}</li>"
+			ulSummary = @getUlFromCodeArray(linkedEntities, codeKey, nameKey, link)
+			entitySummary += ulSummary
+		return entitySummary
+	
+	summarizeDryRunResults: (data) ->
+		# Returns html string with summary of dependency check results
+		
+		dependencies = data.dependencies
+		changesToLotSummary = "<h3>Changes to this #{@lotLabel.toLowerCase()}</h3>"
+		changesToLotSummary += "<ul>"
+		changesToLotSummary += "<li>Parent will be updated from #{data.originalParentCorpName} to #{@.newParentCorpName}</li>"
+		changesToLotSummary += "<li>#{@lotLabel} Molecular Weight will be recalculated</li>"
+		changesToLotSummary += "<li>#{@lotLabel} Number will be updated from #{data.originalLotNumber} to #{data.newLot.lotNumber}</li>"
+		changesToLotSummary += "<li>#{@lotLabel} Name will be: #{data.newLot.corpName}</li>"
+
+		# Get linked experiments summary
+		experimentSummary = @summarizeLinkedCodeTable(dependencies.linkedExperiments, "Dependent experiment results which will be moved to #{@.newParentCorpName}", "/entity/edit/codeName/")
+		changesToLotSummary += experimentSummary
+
+		# Get linked lots summary
+		lotSummary = @summarizeLinkedCodeTable(dependencies.linkedLots,"Remaining Lots On Parent", @lotLink)
+
+		# Get linked container summary
+		containerSummary = @summarizeLinkedCodeTable(dependencies.linkedContainers,"Dependent Inventory Results")
+		changesToLotSummary += containerSummary
+
+		changesToLotSummary += "</ul>"
+
+		errorSummary = "<h3>Errors</h3><ul>"
+		@$('.reparentLotButton').show()
```

Comment:
nit: Kindof funny how this show button line is in the middle of the lines that construct `errorSummary`

Reply:
Yeah, I realize now it's because there are no error cases if the service returns successfully.  I added a note about this.